### PR TITLE
tmpmail: add page

### PR DESCRIPTION
**Pull request by GianniBYoung**
- [X] The page (if new), does not already exist in the repo.
- [X] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [X] The page has 8 or fewer examples.
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [X] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [X] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/tmpmail.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+- Open user specified message:
+
+`tmpmail {{email_id}}`
+
+- View email as raw text without HTML tags:
+
+`tmpmail --text`
+
+- Open email with given browser (default is w3m):
```
Comment (navarroaxel) [inline pages/common/tmpmail.md:26]:
```suggestion
- Open email with a specific browser (default is w3m):
```

File: pages/common/tmpmail.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+- List messages and their numeric ID:
+
+`tmpmail`
+
+- View a temporary email's most recent email:
+
+`tmpmail --recent`
+
+- Open user specified message:
```
Comment (navarroaxel) [inline pages/common/tmpmail.md:18]:
```suggestion
- Open a specific message:
```

File: pages/common/tmpmail.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+- Create a temporary email:
+
+`tmpmail --generate`
+
+- List messages and their numeric ID:
+
+`tmpmail`
+
+- View a temporary email's most recent email:
```
Comment (navarroaxel) [inline pages/common/tmpmail.md:14]: Is this for received emails right? (no sent)
```suggestion
- Display the most recent received email:
```

File: pages/common/tmpmail.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
+# tmpmail
+
+> A temporary email right from your terminal written in POSIX sh.
+> More information: <https://github.com/sdushantha/tmpmail>.
+
+- Create a temporary email:
```
Comment (navarroaxel) [inline pages/common/tmpmail.md:6]: I'm not sure if we should call this `inbox`:
```suggestion
- Create a temporary inbox:
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Can we send emails with this `tmpmail`?

**Comment by GianniBYoung** [comment]
> Can we send emails with this `tmpmail`?

No we can't, it is mainly meant for reciving verification emails when signing up for a service.

**Review by navarroaxel — APPROVED** [review]

**Review by sahilister — APPROVED** [review]
Thanks @GianniBYoung for your contribution and welcome to tldr-pages.

**Pull request merged**